tests/Contracts/CErc20Harness.sol

```diff
@@ -160,7 +160,7 @@ contract CErc20DelegateHarness is CErc20Delegate {
     }
 
     function harnessSetFuseFeeFresh(uint newFuseFeeMantissa) public returns (uint) {
-        return _setFuseFeeFresh(newFuseFeeMantissa);
+        return 0; // _setFuseFeeFresh(newFuseFeeMantissa);
```

Comment:
not sure if that's related to your changes @Joeysantoro (like the error code refactor), but a comment to explain this would have been useful

Reply:
there's no `_setFuseFeeFresh` function anymore on `CToken.sol`. so that's why this error occurs